pkg/service/schema.go

```diff
@@ -342,12 +376,16 @@ type ObjectPropertySchema struct {
 // MarshalJSON provides functionality to marshal an ObjectPropertySchema to JSON
 func (o ObjectPropertySchema) MarshalJSON() ([]byte, error) {
 	type objectPropertySchema ObjectPropertySchema
+	ops := objectPropertySchema(o)
+	if ops.Additional == nil {
+		ops.Additional = &falsePropertySchema{}
+	}
```

Comment:
Oh... Go...

In JSON schema, `additionalProperties` either needs to be `false` or another schema object. Go is strongly typed and can't have something be a boolean or an object...

What we do here is if `Additional == nil`, we replace it with a dummy `falsePropertySchema` whose custom marshaling will just produce the JSON value `false`.